### ISSUE TITLE
initial index shows wrong icon but the currect page

### ISSUE THREAD

**Issue by SaliBhdr** [rafalbednarczuk/curved_navigation_bar#39]
On application start the initial index is showing the wrong icon.
I set the initial index to 2 but it shows the icon from index 0

**Comment by reckers** [comment]
If this is still a problem, I'd check out gepd's comment here [How to animate based on the swiping movements of the tabbarview?](https://github.com/rafalbednarczuk/curved_navigation_bar/issues/21).

**Comment by Mrcel97** [comment]
I've wrongly followed this [documentation](https://flutterawesome.com/a-flutter-package-for-easy-implementation-of-curved-navigation-bar/) where the basic example imports in YAML file:
```
dependencies:
  curved_navigation_bar: ^0.1.2 #latest version
```

Then I've checked closed issues, specifically this one #13. Where it says the problem is solved since 30 Mar. After finding this package releases [here](https://pub.dev/packages/curved_navigation_bar#-changelog-tab-) I've decided to modify the import to the latest version (0.3.1) even still having the ``` ^ ``` before the version.

```
dependencies:
  curved_navigation_bar: ^0.3.1
```

This solved my middle icon (index = 1) display issue.
